asst1/asst1.py

```diff
@@ -0,0 +1,104 @@
+from collections import Counter
+
+def swapchars(string_thing):
+    # initialize a counter and count each letter's occurrence
+    count = Counter()
```

Comment:
I think that my goal in this case was being able to filter out non-alphabetic characters, though if there is a better way to do so, my bad!